README.md

```diff
@@ -1,16 +1,19 @@
 # cloudstart
-## R Markdown documents for RStudio Cloud ☁ to accompany tidymodels articles
 
-This repository contains R Markdown documents to accompany the [Get Started](https://www.tidymodels.org/start/) articles on https://www.tidymodels.org/:
+## R Markdown documents for RStudio Cloud ☁️ to accompany tidymodels articles
 
-1. [Build a model](https://www.tidymodels.org/start/models/)
-2. [Preprocess your data with recipes](https://www.tidymodels.org/start/recipes)  
-3. [Evaluate your model with resampling](https://www.tidymodels.org/start/resampling)  
-4. [Tune model parameters](https://www.tidymodels.org/start/tuning/)   
-5. [A predictive modeling case study](https://www.tidymodels.org/start/case-study/)
+This repository contains R Markdown documents to accompany the five [Get Started](https://www.tidymodels.org/start/) articles on https://www.tidymodels.org/.
```

Comment:
I think we only need 1 link to tidymodels.org here, I would probably just pick one?

Reply:
ok removed!